api/src/main/java/org/openmrs/DrugIngredient.java

```diff
@@ -9,21 +9,39 @@
  */
 package org.openmrs;
 
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
 /**
  * DrugIngredient
  */
-public class DrugIngredient extends BaseOpenmrsObject implements java.io.Serializable, OpenmrsObject {
```

Comment:
Is there any reason why you dropped only `java.io.Serializable` but not `OpenmrsObject`?